server/sockets/list.js

```diff
@@ -107,11 +109,21 @@ const updateItemState = (socket, itemClientLocks) => {
 
 const updateItem = socket => {
   socket.on(ItemActionTypes.UPDATE_SUCCESS, data => {
-    const { listId } = data;
+    const { listId, userId } = data;
+    const sanitizedUserId = sanitize(userId);
 
-    socket.broadcast
-      .to(listChannel(listId))
-      .emit(ItemActionTypes.UPDATE_SUCCESS, data);
+    User.findById(sanitizedUserId).then(user => {
+      if (user) {
```

Comment:
Please do not hide the error when the user is not found - please let it throw.

Reply:
but we are intentionally not catching errors within sockets logic, as sockets are not the main functionality of the app. 

If there is an error we do not handle them in any way, because it will not affect app functionality.

Reply:
I think you should at least log it. Anyway, if it is by design, then it is OK for me.